modules/ROOT/pages/npm-projects.adoc

```diff
@@ -33,9 +33,71 @@ The location of the main {productname} script will be: `+node_modules/tinymce/ti
 * Using a webserver route, or
 * Copying the `+tinymce+` directory to a public folder using a build tool such as Gulp or Webpack.
 
+[[install-premium-plugins]]
+== Install premium plugins
+
+The `+tinymce-premium+` NPM package provides all premium plugins, skins, themes, icons, and language packs for commercial TinyMCE customers.
+
+[IMPORTANT]
+====
+The `+tinymce-premium+` package requires a commercial license. Ensure you have a valid license key before using premium plugins. For information on obtaining a license key, see: xref:license-key.adoc[License Key].
+====
+
+=== Prerequisites
+
+Before installing premium plugins, ensure you have:
+
+* The core `+tinymce+` package installed (see above)
+* A valid commercial license key
+* Node.js and npm installed
+
+=== Install the premium plugins package
+
+Install the `+tinymce-premium+` package using npm:
+
+[source,sh]
+----
+npm install tinymce-premium@^8.3
+----
+
+[NOTE]
+====
+The `+tinymce-premium+` package is available for {productname} 8.3 onwards and is currently only published to the NPM registry (Yarn can also install from NPM). The package version must align with the `+tinymce+` package version—always use compatible versions of both packages (e.g., both at 8.3.0 or later).
+====
+
+=== Package contents
+
+The `+tinymce-premium+` package includes:
+
+* All premium plugins
+* License key manager plugin
+* Premium skins, themes, icons, and language packs for premium plugins
```

Comment:
```suggestion
* Premium skins, icons, and language packs for premium plugins
```

Reply:
Resolved in [this commit](https://github.com/tinymce/tinymce-docs/pull/3928/commits/85a88c5d10dc4c8e3a2b2780fc45778b20ed0be7)